lp-tool/main.go

```diff
@@ -91,15 +94,28 @@ func parseFlags() (Config, error) {
 
 	flag.Parse()
 
-	if config.InputFile == "" || len(config.Tags) == 0 {
-		return config, fmt.Errorf("input file or tags not specified")
+    // Check for invalid options.
+	if config.InputFile == "" {
+		return config, fmt.Errorf("empty input file specified")
```

Comment:
Nitpick, so only change if you're making other changes:
This reads to me that the file itself is empty, even though what you have is technically correct, I would suggest something like:

```suggestion
		return config, fmt.Errorf("no input file specified")
```

Reply:
will fix